examples/list_audiences/Ballerina.toml

```
@@ -0,0 +1,8 @@
+[package]
+org = "ballerinax"
+name = "list_account_exports"
+version = "0.1.0"
+distribution = "2201.8.4"
```

Why is this running on 8.4? 
```suggestion
distribution = "2201.12.0"
```